### PR TITLE
Added ﻿HOL - Experimentation and A/B Testing with Optimizely

### PR DESCRIPTION
**Pull request by bernarden**

### CODE REVIEW

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+**III. Setup second variation in Optimizely** In this step you will create another variation of PartsUnlimited website with more customizations.
+
+**IV. Updating settings of your experiment** In this step you will look through and update settings for the experiment created in this lab.
+
+**V. Viewing results of your experiment** In this step you will see where to find your experiment's results, how it looks and how to export that data.
+
+
+
+### I: Setup first variation in Optimizely
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Is "Task 1" better?

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+**1.** Create an account with [Optimizely](https://www.optimizely.com/). Login after confirming your email address and entering password.
+
+**2.** After being navigated to a "Welcome to Optimizely" page, click on "Start with Web".
+
+![](<media/1.png>)
+
+**3.** On the next page enter your `production` site url and click on "Create Project". This will navigate you to the editor for this experiment.
+
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Use our PartsUnlimited URL as an example

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+![](<media/2.png>)
+
+**4.** Dismiss tutorial and let's define the first variation where we will change the styling of "Shop Now" button. Click on `<edit code>` button in the right bottom corner, enter the following code and click on "Apply" afterwards:
+
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Might need to highlight "Edit Code"

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+    $(".carousel-link").css({"background-color":"#1c367c"});
+
+![](<media/3.png>)
+
+**5.** Save this variation by clicking on "Save now". Let's click on "Start Experiment" to activate this experiment. Close the first pop-up. The second pop-up - "Diagnostic Report" will indicate that a particular snippet of code has to be added to the project so that Optimizely can redirect users to the correct variation of the website. Take a note of this line of code.
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Might need to highlight "Save now"

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+
+    git push
+
+**3.** Once the CI build and deployment to `dev` slot is completed, push your changes to the `staging` slot and finally `production` environment.
+
+
+
+### III: Setup second variation in Optimizely
+
+**1.** Click on "+ Add Variation", ignore a warning about making changes while the experiment is running.
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: How to go back and add the second variation, such as clicking on "Edit" button.

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+![](<media/11.png>)
+
+**3.** In this variation background color of all "Shop Now" buttons will be changed.
+
+  * Change background color of the "Shop Button" in the carousel the same way as in variation 1.
+
+  * If you scroll down you will see that all elements in "Arrivals" and "Top selling" groups also contain their own "Shop Now" buttons. Their color hasn't been changed because `.carousel-link` class is not used for these buttons.
+
+    ![](<media/12.png>)
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: What's the code for at the bottom of the image?

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+  3. Next, adjust the chance of seeing a particular variation of the website for the users included in this experiment.
+  4. Click on "Apply".
+
+      ![](<media/18.png>)
+
+  >  **Note:** If 0% are included then none of the users will see any changes.
+
+**2.** Experiment's goals:
+  1. Click on "Edit" next to "Goals".
+  2. Set "Shop Button" goal as a primary goal for this experiment.
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Didn't see the "Shop Button" goal.

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+      ![](<media/19.png>)
+
+**3.** Audiences:
+  1. Click on "Edit" next to "Audiences".
+  2. Click on "Create a New Audience"
+
+      ![](<media/20.png>)
+
+  3. Enter Name for your audience. Drag "Device" condition to "Audience Conditions" or any other condition you want to use and set it up as you wish. Click on "Save"
+
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Save or Done?

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+      ![](<media/21.png>)
+      >**Note:** It's possible to create logical expressions using these conditions by dragging them into "and" or "or" sections.
+
+  4. Click on "Done".
+
+**4.** URL Targeting:
+  1. Click on "Edit" next to "URL Targeting".
+  2. Let's add `dev` and `staging` slots to this experiment.
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Sample URL?

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+        ![](<media/22.png>)
+
+**5.** Scheduler:
+  1. Click on "Edit" next to "Scheduler".
+  2. Select time when experiment will start
+  3. Select time when experiment will end
+  4. Select time zone.
+  3. Click on "Save"
+
+        ![](<media/23.png>)
```
Comment (steven2000nz) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Image missing?

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 1 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Updated this.

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 3 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Done.

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 4 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: It's already highlighted.

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 5 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Done

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 6 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: this code was generated by step 2 and substep 1 of step 3 in task 3

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 7 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Added the required step

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 8 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: There are multiple pages. Made this more clear.

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 10 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: Fixed

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 9 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: This is specified in screenshot

File: docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md (hunk 2 above)
Comment (bernarden) [inline docs/HOL-Experimentation_and_AB_Testing_with_Optimizely/README.md]: This is specified in screenshot

**Pull request merged**